index.html

```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+    <title>portofolio</title>
+</head>
+<body>
+    
+    <div class="container">
+        <div class="card">
+        <header>
+            <div class="fcontainer">
+                <a class="logo">Saba</a>
+                <nav id="items">
+                  <i class="fa fa-bars"></i>
+                </nav>
+              </div>
+          </header>
+           <div class="fcontent">
+            <h1 class="name">I'M Saba Ahmad <br> Glad to see <br> you!</h1>
+            <p class="paragraph1">
+              I'm a software developer! I can help you  build a product , feature or website look  through some of my work  experience ! if you like what you see and  have a project you need coded, don't hesitate to contact me
+          </p>
+           </div>
+           <div class="connect">
+            <h3>LET'S  CONNECT</h3>
+        </div>
+        <div class="icons">
+            <ul> 
+           <li><img src="image/image8.png" alt="twitter"></li> 
+          <li><img src="image/imge5.png" alt="twitter"></li> 
+          <li><img src="image/image9.png" alt="twitter"></li> 
+          <li><img src="image/image6.png" alt="twitter"></li>
+          <li><img src="image/image7.png" alt="twitter"></li> 
+           </ul>
+            </div>
+</div>
+        
+        <section>
```

Comment:
Please note that you needed have your projects inside a container styled using `display:grid` as part of the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md
![Uploading Screenshot 2022-09-23 at 16.28.08.png…]()
). Currently your cards container don't have the `display:grid` property.